manager/src/main/java/org/apache/hertzbeat/manager/component/alerter/DispatcherAlarm.java

```diff
@@ -113,6 +118,13 @@ public void run() {
                         alertStoreHandler.store(alert);
                         // Notice distribution
                         sendNotify(alert);
+                        // Execute the plugin
+                        List<Object> beans = pluginConfig.getBean();
+                        for (Object bean : beans) {
+                            if (bean instanceof Plugin) {
```

Comment:
Is it possible to use generics here to avoid type conversion?

Reply:
Please help me modify it.